src/components/templates/PosterHall/PosterHall.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import { GenericVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { enterVenue } from "utils/url";
+
+import { usePosters } from "hooks/posters";
+
+import { PosterPreview } from "./components/PosterPreview";
+import { Search } from "./components/Search";
+
+import "./PosterHall.scss";
+
+export interface PosterHallProps {
+  venue: WithId<GenericVenue>;
+}
```

Comment:
I think I saw a non-generic poster venue in the types. Is that for this? Or is that for the poster venue itself?

Reply:
There are two different templates: `PosterHall`(no need to create a separate Template Type) this^. And a PosterPage, which has some unique properties and is a separate template